workitem/workitem_repository.go

```diff
@@ -166,10 +294,20 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, typeID string, fiel
 	if err != nil {
 		return nil, errors.NewBadParameterError("type", typeID)
 	}
+
+	// The order of workitems are spaced by a factor of 1000.
+	pos, err := r.LoadHighestOrder()
+	if err != nil {
+		return nil, errors.NewInternalError(err.Error())
+	}
+	pos = pos + orderValue
+	fields[SystemOrder] = pos
```

Comment:
@aslakknutsen does your concern still hold true or can you please elaborate a bit?